File: src/api/Modules/Resumes/Faber.Modules.Resumes.Application/Features/Persons/UpdatePerson/UpdatePersonRequest.cs
namespace Faber.Modules.Resumes.Application.Features.Persons.UpdatePerson;

public record UpdatePersonRequest(
    Guid Id,
    Guid ResumeId,
    string? JobTitle,
    string? Firstname,
    string? Lastname,
    string? Email,
    string? Phone,
    string? Country,
    string? City,
    string? Street,
    string? PostCode,
    string? Nationality,
    DateOnly? DateOfBirth,
    string? DrivingLicense);

File: src/web/faber-app/src/app/modules/resumes/builder/person/update-person-request.ts
export interface UpdatePersonRequest {
  jobTitle: string;
  firstname: string;
  lastname: string;
  email: string;
  phone: string;
  country: string;
  city: string;
  street: string;
  postCode: string;
  nationality: string;
  dateOfBirth: string | null;
  drivingLicense: string;
}
